packages/groq-builder/package.json

```diff
@@ -54,5 +61,8 @@
   },
   "publishConfig": {
     "provenance": true
+  },
+  "dependencies": {
+    "groq-builder": "^0.1.2"
```

Comment:
@scottrippey `groq-builder` can't depend on itself, no?

Reply:
Thanks for catching this!